### PR TITLE
Split CSS into multiple page-specific files

### PR DESCRIPTION
**Pull request by n-wach**
Continuation of the webapp directory restructure.

See design doc: https://docs.google.com/document/d/12t7FDPDpTpHzrQBJ3T8qdmGKGrPi3lr2-UPmoc2ISHI/edit#heading=h.5lbktxp4m9ll

Note: this is blocking several PRs and should be merged quickly

### CODE REVIEW

File: src/main/webapp/list/list.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/* List page */
```
Comment (billie-cat) [inline src/main/webapp/list/list.css:1]: nit, can this cc file be lecture-list.css, list is very generic.

Comment (n-wach) [reply src/main/webapp/list/list.css]: Done!

File: src/main/webapp/view/lecture-view.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  height: 200px;
+  max-height: 200px;
+  overflow: hidden;
+}
+
+#new-comment-form {
+  width: 100%;
+}
+
+/* Transcript */
```
Comment (billie-cat) [inline src/main/webapp/view/lecture-view.css:34]: Only add these once they are needed.

Comment (n-wach) [reply src/main/webapp/view/lecture-view.css]: Removed

File: makefile [hunk 3]
```diff
@@ -14,7 +14,7 @@ pretty: node_modules
 
 validate: node_modules
 	$(HTML_VALIDATE) "src/main/webapp/**/*.html"
-	$(CSS_VALIDATOR) src/main/webapp/*.css
+	$(CSS_VALIDATOR) src/main/webapp/*.css src/main/webapp/view/*.css src/main/webapp/list/*.css
```
Comment (f-camp) [inline makefile:17]: You would rename this as well to lectures/*.css right?

Comment (n-wach) [reply makefile]: Done!

**Pull request merged**